docs/intro.md

```diff
@@ -10,44 +10,43 @@ Make stream processing easier!
 
-On the other hand, It can be challenge for enterprises to use Flink & Spark if there is no professional management platform for Flink & Spark tasks during the deployment phase. StreamPark provides such a professional task management platform, including task development, scheduling, interactive query, deployment, operation, maintenance, etc.
+On the other hand, It can be challenge for enterprises to use Flink & Spark if there is no professional management platform for Flink & Spark tasks during the deployment phase. StreamPark provides such a professional task management platform as described above.
 
 ## 🎉 Features
 
 * Apache Flink & Apache Spark application development scaffold
-* Support multiple versions of Flink & Spark
+* Supports multiple versions of Flink & Spark
 * Wide range of out-of-the-box connectors
 * One-stop stream processing operation platform
-* Support catalog、olap、streaming-warehouse etc.
-* ...
+* Supports catalog、olap、streaming-warehouse, etc.
 
 ## 🏳‍🌈 Architecture of Apache StreamPark™
 
-The overall architecture of Apache StreamPark is shown in the following figure. Apache StreamPark consists of three parts, they are streampark-core and streampark-console.
+The overall architecture of Apache StreamPark is shown in the following figure. Apache StreamPark has two parts, `streampark-core` and `streampark-console`.
 
 ![StreamPark Archite](/doc/image_en/streampark_archite.png)
 
 ### 1️⃣ streampark-core
 
-The positioning of `streampark-core` is a framework uesd while developing, it focuses on coding development, regulates configuration files, and develops in the convention over configuration guide.
+`streampark-core` is a framework uesd during development. It supports on coding development, regulates configuration files, and follows the 'convention over configuration' guide.
 
-`streampark-core` provides a development-time RunTime Content and a series of out-of-the-box Connectors. Cumbersome operations are simplified by extending DataStream-related methods and integrating DataStream and Flink SQL API. Development efficiency and development experience will be highly improved because users can focus on the business.
+`streampark-core` provides development-time RunTime Content and a series of out-of-the-box Connectors. Cumbersome operations are simplified by extending DataStream-related methods and integrating DataStream and the Flink SQL API. This improves development efficiency and developer experience, because users can focus on the business logic.
```

Comment:
```suggestion
`streampark-core` provides development-time Runtime Content and a series of out-of-the-box Connectors. Cumbersome operations are simplified by extending DataStream-related methods and integrating DataStream and the Flink SQL API. This improves development efficiency and developer experience, because users can focus on the business logic.
```